src/components/card/_card.scss

```diff
@@ -5,6 +5,12 @@
   &__link {
     display: flex;
     flex-direction: column;
+    margin-bottom: 1rem;
+    width: fit-content;
+  }
+
+  &__remove--margin {
+    margin-bottom: 0;
   }
```

Comment:
```
  &__link {
    display: flex;
    flex-direction: column;
    margin-bottom: 1rem;
    width: fit-content;

    & > .ons-card__title {
      margin-bottom: 0;
    }
  }
```
I would do something like this instead because inside the link will always be a title. Then we can remove the `ons-card__remove--margin` class from the html too.

Reply:
Change has been made, I had tried this previously, but my styles wasn't applying so had to go with other approach, however this code is clearly working :)